misc/vasp/python/processors/chgcarsource.py

```diff
@@ -0,0 +1,100 @@
+# Name: ChgcarSource
+
+import inviwopy as ivw
+import ivwdataframe as df
+import atomdata
```

Comment:
Was talking with @martinfalk about how we could ensure that these python dependencies are installed. We should have a discussion about that when we come back.

Reply:
that might be good, although all this is local modules so they should always be there....